tests/gradients/test_gradient_transform.py

```diff
@@ -456,8 +456,8 @@ def circuit(x):
 
         # the gradient function can be called with different shot values
         grad_fn = qml.gradients.param_shift(circuit)
-        assert grad_fn(x).shape == (1, 1)
-        assert grad_fn(x, shots=[(1, 1000)]).shape == (1000, 1)
+        assert grad_fn(x).shape == ()
+        assert grad_fn(x, shots=[(1, 1000)]).shape == (1000,)

```

Comment:
very nice! I remember this annoying me when I first wrote this test, but I didn't know how to fix it